<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMeasurementWidget</class>
 <widget class="QWidget" name="AddMeasurementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Value* :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Measurement type* :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="measurementTypeComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="materialComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Material* :</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string>Picture</string>
   </property>
  </widget>
  <widget class="QLabel" name="commentsLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
